Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/General/Scripts/Utils/LoadProfilesOnStartup.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
nit, this has been reduced:

// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

Reply:
Updated license header.